setup.py

```diff
@@ -14,7 +14,7 @@
 extras_require: dict[str, list[str]] = {
     "array": ["numpy >= 1.18"],
     "bag": [],  # keeping for backwards compatibility
-    "dataframe": ["numpy >= 1.18", "pandas >= 1.0"],
+    "dataframe": ["numpy >= 1.18", "pandas >= 1.3"],
```

Comment:
I'm not sure if we want to bump the minimum pandas version here. 1.3 came out ~9 months ago. Is there a way to make these changes backwards compatible?